docs/eventing/sources/pingsource.md

```diff
@@ -98,13 +98,13 @@ EOF
 {{< /tab >}}
 
 {{% tab name="kn" %}}
-
+Notice that the namespace is specified in two places in the command in `--namespace` and the `--sink` hostname
 ```shell
 kn source ping create test-ping-source \
-  --namespace pingsource-example
+  --namespace pingsource-example \
   --schedule "*/1 * * * *" \
   --data '{"message": "Hello world!"}' \
-  --sink http://event-display.svc.cluster.local
+  --sink http://event-display.pingsource-example.svc.cluster.local
```

Comment:
Any particual reason why the URL is used for the sing and e.g. not `--sink event-display` (if targeting a Knative service within the same namespace) ?

Reply:
Also, you can skip the first namespace argument if you just want to create the source in the _current_ namespace (which also save the step of creating a particular namespace). imo its ok to assume that the user know that commands work on a _current namespace_ if no namespace is provided (an idiom known from `kubectl`)

Reply:
The URL format should be only used if there is no better way to address the sink (e.g. across namespaces). That said, it would be of course nice to support a namespace in the cli sink-syntax, too.

Reply:
I tried to figured out how to specify the service from CLI help it showed <kind>:name 

The service is not ksvc just a normal k8s service. 

I think tried service:event-display and svc:event-display and I remember getting an error.

Reply:
About specifying the namespace in the kubectl is following the example on creating the namespace at the start then using the namespace in every kubectl command. 

We don't want to introduce complexity to the new user on using the syntax of switching namespace context (ie kubectl config set-context --current --namsoace ns)

I don't design this example but all.examples in eventing use the pattern so we should be consistent

Reply:
Sorry, overlooked that it is a standard service.  Yes, then URL is the best option if you don't want to include a mapping for a k8s in the kn configuration. We have not added regular K8s services to the standard mapping because it's outside of Knative. However if people have a strong feeling to include it nevertheless (and e.g. connect it to the prefix  `svc:`), then we could consider this. It's just that it would lead to an error message on non-k8s implementations of Knative (e.g. Cloud Run)